recipes/imagededup/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "imagededup" %}
+{% set version = "0.2.1" %}
+{% set sha256sum = "5bd41845550cf4c4dd4431a3932dcb8beca138ffee895cd1183c9d517ec9fb52" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [(win and py27) or (osx and py27) or (linux and py27)]
```

Comment:
```suggestion
  skip: true  # [py27]
```

Reply:
Resolved I added [py<36] instead because we only want to build it for py36 & py37 or higher in the future like Python 3.8.